ai-ml/trainium-inferentia/variables.tf

```diff
@@ -1,10 +1,11 @@
 variable "name" {
   description = "Name of the VPC and EKS Cluster"
-  default     = "trainium-inferentia"
```

Comment:
keep the same name is its being used in multiple blueprints

Reply:
reverted